docs/user_docs/upgrade/upgrade-with-helm/upgrade-kubeblocks-to-0.9.md

```diff
@@ -56,39 +46,40 @@ KubeBlocks 0.9 is compatible with KubeBlocks 0.8 APIs, but compatibility with AP
    kubectl delete opsdefinitions.apps.kubeblocks.io kafka-quota kafka-topic kafka-user-acl switchover
    ```
 
-3. Install CRD.
+3. Install the StorageProvider CRD before the upgrade.
 
-    To reduce the size of Helm chart, KubeBlocks v0.8 removes CRD from the Helm chart. Before upgrading, you need to install CRD.
+    If the network is slow, it's recommended to download the CRD YAML file on your localhost before further operations.
 
-    ```shell
-    kubectl replace -f https://github.com/apecloud/kubeblocks/releases/download/v0.9.0/kubeblocks_crds.yaml
+    ```bash
+    kubectl create -f https://github.com/apecloud/kubeblocks/releases/download/v0.9.0/dataprotection.kubeblocks.io_storageproviders.yaml
```

Comment:
Why install this CRD only?

Reply:
> Why install this CRD only?

The upgrade process only requires installing this CRD and the process will handle the others.